src/js/utils/Util.js

```diff
@@ -249,6 +250,30 @@ const Util = {
       search: aElement.search,
       username: aElement.username
     };
+  },
+
+  isMobile() {
```

Comment:
Shall we move these to `ViewportUtil` or similar? (Both/all new functions)

Reply:
Sounds good.